homeassistant/components/switch/broadlink.py

```diff
@@ -209,7 +263,12 @@ def should_poll(self):
     @property
     def is_on(self):
         """Return true if device is on."""
+        self.update()
```

Comment:
Revert this

Reply:
If sub module override `update()` and need to call the for update `state`

Reply:
I think it's necessary to call it here.
I have a easy way to check this ,
Just keep the switch on during HA reboot,
and the state is always off without this.

Reply:
This is not the correct way of doing it, so it will not be accepted.
So for what type of device to you experience this?

Reply:
i got this device named `SP mini 3` or `SP 3 mini`
I think all the `SP2_TYPES` has the same problem.
![sp3mini](https://i.ebayimg.com/images/g/XnUAAOSwFYxZxglF/s-l200.jpg)

Reply:
Ok, does it never update without this change?

Reply:
emmm...  it's  weird.
i tested again and found it works without this change.
maybe i upgrade homeassitant for `0.60.1` to `0.61.1` and fixed it.
by the way, I will remove this change ,and sorry for my mistake.